docs/decisions/0006-make-redux-deps-optional.rst

```diff
@@ -0,0 +1,51 @@
+Consumers of frontend-platform that don't use redux are still required to install redux dependencies
+==========================================
+
+Status
+------
+
+Review
+
+Context
+-------
+
+Some consumers of frontend-platform, such as frontend-app-learner-portal-enterprise, no longer use redux or react-redux.
+However, they are still required to install these dependencies due to the usage of react-redux in AppProvider, and the fact that
```

Comment:
Might be expanding on what required means, describing what happens currently if you uninstall these dependencies today, i.e. fails to build due to the `import` of `react-redux`.